crates/host/src/providers/httpserver/src/main.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    println!("Hello, world!");
+}
```

Comment:
I assume this is TODO once wasi-http is implemented?

Reply:
No, this definitely should not have been here. 😅